buildscripts/set_tensorflow_bazelrc.sh

```diff
@@ -37,6 +48,11 @@ build:xla --define with_xla_support=true
 build --config=xla
 build:opt --copt="${OPTION_1}"
 build:opt --copt="${OPTION_2}"
+build:opt --copt="${OPTION_3}"
+build:opt --copt="${OPTION_4}"
+build:opt --copt="${OPTION_5}"
+build:opt --copt="${OPTION_6}"
+build:opt --copt="${OPTION_7}"
 build:opt --host_copt="${OPTION_1}"
```

Comment:
Do OPTION_3-OPTION_7 also need to be added for `host_copt`?

Reply:
I think host_copt settings are for nvcc compilers.